### ISSUE TITLE
ModuleNotFoundError: No module named 'AppKit'

### ISSUE THREAD

**Issue by ggrelet** [rbignon/doctoshotgun#47]
## Description
When a center with available slot is found, I encountered a `ModuleNotFoundError`

## Stacktrace
Here is the last bit of the trace:
```
Center Centre de vaccination covid-19 - Centre Henry Dunant:
– Centre Henry Dunant - Croix Rouge Française... first slot not found :(

Center Centre de vaccination COVID-19 - Gymnase Henry de Montherlant - Paris 16:
– Gymnase Henry de Montherlant... found!
  ├╴ Best slot found: Fri May 28 09:45:00 2021
Traceback (most recent call last):
  File "./doctoshotgun.py", line 468, in <module>
    sys.exit(Application().main())
  File "./doctoshotgun.py", line 454, in main
    if docto.try_to_book(center):
  File "./doctoshotgun.py", line 252, in try_to_book
    if self.try_to_book_place(profile_id, motive_id, practice_id, agenda_ids):
  File "./doctoshotgun.py", line 306, in try_to_book_place
    playsound('ding.mp3')
  File "./doctoshotgun.py", line 30, in playsound
    return _playsound(*args)
  File "/Users/me/.virtualenvs/docto/lib/python3.8/site-packages/playsound.py", line 55, in _playsoundOSX
    from AppKit     import NSSound
ModuleNotFoundError: No module named 'AppKit'
```

## Environment
```
macOS 11.3
Python 3.8.2
pip 21.1
```

**Comment by rbignon** [comment]
Thank you for your report. I've fixed this issue in bb4bb15d08c9a501f3c88ea3759b85bbfc8ed915, you can update the repository.

**Comment by ggrelet** [comment]
Thanks I'll try. For your information `pip install pyobjc` seems to be fixing it. I can make a PR if you want.
